tdi_python/tdiTable.py

```diff
@@ -1815,6 +1815,13 @@ def set_supported_attributes_to_supported_commands(self):
         for i in range(len(attributes_arr)):
             self.supported_commands += self.attributes_type_map(attributes_arr[i])
 
+        # TODO: Enable this when attributes are supported
+        '''
+        for i in range(len(attributes_arr)):
+            if attributes_arr[i] in keys_list:
+                self.supported_commands += self.attributes_type_cls.attributes_dict[attributes_arr[i]]
+        '''
+
```

Comment:
We need uncomment out this code in order to see the attributes list on the table.

Reply:
Yes I need to move code out to target specific code including all the different attribute functions themselves